ceresdbproto_deps/Cargo.toml

```diff
@@ -9,5 +9,5 @@ workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies.ceresdbproto]
-git = "https://github.com/CeresDB/ceresdbproto.git"
-rev = "8e54107e0255429313024822ec85b78561939d00"
\ No newline at end of file
+git = "https://github.com/ShiKaiWi/ceresdbproto.git"
```

Comment:
After https://github.com/CeresDB/ceresdbproto/pull/21 merged, update this repo url and the commit.